### PR TITLE
 Ihor Dvoretskyi now commits for CNCF

### PR DESCRIPTION
**Pull request by idvoretskyi**

### CODE REVIEW

File: company_developers.txt [hunk 1]
```diff
@@ -841,6 +841,7 @@ CNCF:
 	Dan Kohn: dan!dankohn.com, dan!linuxfoundation.org
 	Zachary Corleissen: zacharysarah!users.noreply.github.com from 2017-06-01
 	lukaszgryglicki: lukaszgryglicki!o2.pl from 2017-03-15
+	Ihor Dvoretskyi: ihor!dvoretskyi.com, ihor.dvoretskyi!gmail.com, ihor!linux.com, ihor!cncf.io
```
Comment (lukaszgryglicki) [inline company_developers.txt:844]: @idvoretskyi , can You please provide the date from?
Or maybe just a list of affiliations?
Like for example
Company1 from datefrom1 to dateto2
Company2 from datefrom2 to dateto2
...

I will have to update mapping, those files are machine generated.
Would be great if You just provide You affiliations list in a comment, and I'll update all what is needed.

Comment (dankohn) [inline company_developers.txt:844]: @lukaszgryglicki Bob Wise here should be in his own company, Samsung SDS.

Comment (lukaszgryglicki) [reply company_developers.txt:844]: I've asked for a full list. Once I have it I'll update mappings and regenerate those machine generated files.

**Comment by lukaszgryglicki** [comment]
Resolved via: https://github.com/cncf/gitdm/commit/8eb1be7b0c36aff8c9a4125d286164a67e8c078c

**Pull request closed without merging**